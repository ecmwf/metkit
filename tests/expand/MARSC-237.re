retrieve,
area=90/-180/-90/180,class=od,date=2024-01-01,expver=1,grid=0.25/0.25,levtype=sfc,padding=0,param=mslp,step=0/to/120/by/3,stream=oper,time=00:00,type=fc,
    target="reference.Hx2aUM.data" 