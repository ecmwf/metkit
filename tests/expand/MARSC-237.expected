RETRIEVE,
    CLASS      = OD,
    TYPE       = FC,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 151,
    DATE       = 20240101,
    TIME       = 0000,
    STEP       = 0/3/6/9/12/15/18/21/24/27/30/33/36/39/42/45/48/51/54/57/60/63/66/69/72/75/78/81/84/87/90/93/96/99/102/105/108/111/114/117/120,
    DOMAIN     = G,
    TARGET     = reference.Hx2aUM.data,
    RESOL      = AUTO,
    AREA       = 90/-180/-90/180,
    GRID       = .25/.25,
    PADDING    = 0 